docs/EN_US/ContainerizedHPCC/ContainerizedMods/AzureDeployChapter.xml

```diff
@@ -0,0 +1,645 @@
+# remoteClients:
+# - name: myclient
+#   organization: mycompany
+  service:
+    ## port can be used to change the local port used by the pod. If omitted, the default por
+    port: 8888
+    ## servicePort controls the port that this service will be exposed on, either internally 
+    servicePort: 8010
+    ## Specify visibility: local (or global) if you want the service available from outside  
+externally, while eclservices is designed for internal use.
+    visibility: local
+    ## Annotations can be specified on a service - for example to specify provider-specific i
+-balancer-internal-subnet
+    #annotations:
+    #  service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "mysubnet"
+    #  The service.annotations prefixed with hpcc.eclwatch.io should not be declared here. T
+    #  in other services in order to be exposed in the ECLWatch interface. Similar function c
+    #  applications. For other applications, the "eclwatch" inside the service.annotations sh
+    #  their application names. 
+    #  hpcc.eclwatch.io/enabled: "true"
+    #  hpcc.eclwatch.io/description: "some description"
+    ## You can also specify labels on a service
+    #labels:
+    #  mylabel: "3"
+    ## Links specify the web links for a service. The web links may be shown on ECLWatch.
+    #links:
+    #- name: linkname
+    #  description: "some description"
+    #  url: "http://abc.com/def?g=1"
+    ## CIDRS allowed to access this service.
+    #loadBalancerSourceRanges: [1.2.3.4/32, 5.6.7.8/32]
+  #resources:
+  #  cpu: "1"
+  #  memory: "2G"
+- name: eclservices
+  application: eclservices
+  auth: none
+  replicas: 1
+  service:
+    servicePort: 8010
+    visibility: cluster
+  #resources:
+  #  cpu: "250m"
+  #  memory: "1G"
+- name: eclqueries
+  application: eclqueries
+  auth: none
+  replicas: 1
+  service:
+    visibility: local
+    servicePort: 8002
+    #annotations:
+    #  hpcc.eclwatch.io/enabled: "true"
+    #  hpcc.eclwatch.io/description: "Roxie Test page"
+    #  hpcc.eclwatch.io/port: "8002"
+  #resources:
+  #  cpu: "250m"
+  #  memory: "1G"
+- name: esdl-sandbox
+  application: esdl-sandbox
+  auth: none
+  replicas: 1
+  service:
+    visibility: local
+    servicePort: 8899
+  #resources:
+  #  cpu: "250m"
+  #  memory: "1G"
+- name: sql2ecl
+  application: sql2ecl
+  auth: none
+  replicas: 1
+# remoteClients:
+# - name: sqlclient111
+  service:
+    visibility: local
+    servicePort: 8510
+  #domain: hpccsql.com
+  #resources:
+  #  cpu: "250m"
+  #  memory: "1G"
+- name: dfs
+  application: dfs
+  auth: none
+  replicas: 1
+  service:
+    visibility: local
+    servicePort: 8520
+  #resources:
+  #  cpu: "250m"
+  #  memory: "1G"
+</programlisting>
+
+        <para>Save that ESP portion off into a new file called
+        <emphasis>myesp.yaml.</emphasis> We need to make some modifications to
+        that file then we will use that file to pass those modified values
+        into our deployment.</para>
```

Comment:
We need to modify that file, then use it to override the default values of our deployment.